tutorials/nlp/Joint_Intent_and_Slot_Classification.ipynb

```diff
@@ -76,7 +76,7 @@
     "- **train.tsv/test.tsv** - contain original queries, one per line, and intent number separated by tab. For example: `what alarms do i have set right now\t0`. Intent numbers are according to the intent line in the intent dictionary file (dict.intents.csv) starting from 0. First line of these files contains a header line: `sentence \\tab label`.\n",
     "- **train_slot.tvs/test_slot.tsv** - contain one line per a query, where instead each token there is a number of the token from the slots dictionary file (dict.slots.csv), starting from 0. Last 'out-of scope' token is usually located in the last line of the dictionary. Example: `54 0 0 54 54 12 12` (numbers separated by space). No header line in these files.\n",
     "\n",
-    "Nemo provides **import_dataset.py** converter for few reference datasets (Assistant/Atis/Snips) which converts them to the Nemo data format for the Intent and Slot classification model. If you have your own annotated dataset in a different format, you will need to write a data converter. Possible recommended format for your own annotation, is to have one text file per all examples of one intent. With one line per query in a form like: `did i set an alarm to [alarm_type : wake up] in the [timeofday : morning]`, using brackets to define slot names. This is very similar to the assistant format from this example and you can use it's converter to Nemo format with small changes. \n",
+    "Nemo provides **import_dataset.py** converter for few reference datasets (Assistant/Atis/Snips) which converts them to the Nemo data format for the Intent and Slot classification model. If you have your own annotated dataset in a different format, you will need to write a data converter. Possible recommended format for your own annotation, is to have one text file per all examples of one intent. With one line per query in a form like: `did i set an alarm to [alarm_type : wake up] in the [timeofday : morning]`, using brackets to define slot names. This is very similar to the assistant format from this example and you can use its converter to Nemo format with small changes. \n",
```

Comment:
Do we support this format directly in nemo model?

Reply:
Which one from 3 you are asking? We do have converter mode for each 3 of them to Nemo data format.

Reply:
So here we are suggesting users to create their data in Assitant format and them convert it to NeMo's format?

Reply:
If yes, lets make it more clear here. And are you sure if they gather their data, they can easily use the same convertor? I mean that one is written for that specific dataset and folder structure?

Reply:
I wanted to do that, but there is a small issue. This Assistant data format also it is good, it keeps multiple columns in their original json files. Like Query before manual correction, after manual correction, with annotation, etc. So in my converter, I am using a single columns (separated by ";") for each line of their json files. And although I do like their annotation format it is convoluted with other non relevant data in the same file. So, I did not know how to explain it better to user?

Reply:
This is something that we should work on, but it looks to me it is not ready. So how about dropping this suggestion here for now and come up with a good plan later?

Reply:
This beta version is a good opportunity to reevaluate and redesign the nemo format if you feel like it is not good enough.

Reply:
I sent you my more detailed response in Slack